### ISSUE TITLE
What ever was changed - please let me have it back!

### ISSUE THREAD

**Issue by Vazziera** [tullamods/Bagnon#358]
I used to be able to keep my bags using the blizzard frames so that each bag was seperate but what ever recently was changed I don't seem to be able to do anything but the massive combined view of my bags. I don't like this :( please put the option in to keep the bags blizzard style Thanks

**Comment by Jaliborc** [comment]
Fixed

**Comment by Vazziera** [comment]
I am not sure what exactly you did but I still am not able to do what I was doing before. I have attached a screen shot of what I did before ( I would turn off the guild bank part because that would interfere with my OCD officers organization of the guild bank but see what I mean thru the pictures) and I would use the mass the bags together for my personal bank but leaving the inventory bags more alone as seen in the picture. Is there a way to do this through the addon again? I have tried turning each section of bagnon on.. and off and any combination of them inbetween and still can't do what is in the picture unless I completely turn it off. Thanks for any help!
![wowscrnshot_021615_112838](https://cloud.githubusercontent.com/assets/11305400/6489784/da949352-c266-11e4-905e-09a8e6d19f20.jpg)
